tox.ini

```diff
@@ -18,8 +18,9 @@ deps =
     !minimal: pytest-xdist
     pytest-github-actions-annotate-failures
 commands =
-    !cov: {env:PYTEST_COMMAND} {posargs}
-    cov: {env:PYTEST_COMMAND} {posargs} --cov=plasmapy --cov-report=xml --cov-config={toxinidir}/setup.cfg --cov-append --cov-report xml:coverage.xml
+    !cov: {env:PYTEST_COMMAND} {posargs} -m 'not slow'
+    cov: {env:PYTEST_COMMAND} {posargs} --cov=plasmapy --cov-report=xml --cov-config={toxinidir}/setup.cfg --cov-append --cov-report xml:coverage.xml -m 'not slow'
+    cov-slow: {env:PYTEST_COMMAND} {posargs} --cov=plasmapy --cov-report=xml --cov-config={toxinidir}/setup.cfg --cov-append --cov-report xml:coverage.xml -m 'slow'
```

Comment:
Reading this at first made me wonder if `coverage.xml` would have been overwritten or something, so it's good to hear that Codecov is able to merge coverage reports from multiple tests.

Reply:
Which is why I initially thought we'd have to manually sidestep the issue! I'll keep an eye on the coverage results for the next few days just to be sure.